templates/components/_dropdown_user_desktop.html.twig

```diff
@@ -0,0 +1,91 @@
+<div class="em-dropdown--container em-nav">
+    <div class="l__wrapper">
+        <div class="l__row l__row--top">
+            <div class="em-nav-dropdown__col">
+                <div class="adherent-profile__id l__row b__nudge--bottom">
+                    <div class="name b__nudge--right-nano">{{ app.user }}</div>
+                    {% if is_certified %}
+                        <div class="l__row">
+                            {% include('components/_certification_logo.html.twig') %}
+                        </div>
+                    {% endif %}
+                </div>
+                <ul class="adherent-profile__roles b__nudge--bottom">
+                    {% for role in get_adherent_role_labels(app.user) %}
+                        <li>{{ role }}</li>
+                    {% endfor %}
+                </ul>
+                {#                    <div class="adherent-profile__completion b__nudge--bottom">#}
+                {#                        <div class="text--body text--small b__nudge--bottom-small">#}
+                {#                            <!-- If is not completed -->#}
+                {#                            Complétion du profil : 50%#}
+                {#                            <!-- If is completed TODO#}
+                {#                            👏 Profil complété ! -->#}
+                {#                        </div>#}
+                {#                        <div class="gauge">#}
+                {#                            <span></span>#}
+                {#                        </div>#}
+                {#                        <div class="b__nudge--top-10">#}
+                {#                            <a href="{{ path('app_funnel_general') }}" class="text--body text--small text--blue--dark link--no-decor">Compléter mon profil</a>#}
+                {#                        </div>#}
+                {#                    </div>#}
+                <ul class="">
+                    {% if is_granted('ROLE_FORMATION_SPACE') %}
+                        <li class="space--5-0"><a href="{{ path('app_formation_home') }}" class="text--body text--small link--no-decor">Formation</a></li>
+                    {% endif %}
+                    <li class="space--5-0"><a href="https://aide.en-marche.fr" rel="noopener noreferrer" target="_blank" class="text--body text--small link--no-decor">Aide</a></li>
+                    {% if is_granted('ROLE_REFERENT') or is_granted('ROLE_COREFERENT') %}
```

Comment:
can you add a check on the `ROLE_DELEGATED_REFERENT` please ?

Reply:
you mean adding `or is_granted(ROLE_DELEGATED_REFERENT)` ?

Reply:
yup sorry